src/parser.lalrpop

```diff
@@ -227,6 +227,14 @@ Arg: Argument = {
 	Quoted_String => Argument::Quote(<>),
 	Port => Argument::Port(<>),
 	IPAddr => Argument::IpAddr(<>),
+	Context => Argument::Var(<>),
+}
+
+Context: CascadeString = {
+	// TODO: don't discard the mls range
+	<start: @L> <u: Symbol> ":" <r: Symbol> ":" <t: Symbol> <m: (":" <Symbol>)?> <end: @R> => {
```

Comment:
Just for my learning what are @L and @R?

Reply:
Well, I was hoping to link you to the official documentation, but instead I found this bug that they aren't documented :) https://github.com/lalrpop/lalrpop/issues/343  As far as I can tell the lalrpop project sadly seems abandoned, so hope of doc updates is low.

Anyways, what `@L` and `@R` is provide the location in the file of the left and right of the token next to them.  `@L` provides the leftmost position in the token to the right of it, and `@R` provides the rightmost position in the token to its left.  When used in conjunction, they're how we get our Ranges for CascadeStrings.